src/Microsoft.Health.Fhir.Shared.Api/Features/Formatters/HtmlOutputFormatter.cs

```diff
@@ -67,58 +68,58 @@ public override async Task WriteResponseBodyAsync(OutputFormatterWriteContext co
             EnsureArg.IsNotNull(context, nameof(context));
             EnsureArg.IsNotNull(selectedEncoding, nameof(selectedEncoding));
 
-            var engine = context.HttpContext.RequestServices.GetService<IRazorViewEngine>();
-            var tempDataProvider = context.HttpContext.RequestServices.GetService<ITempDataProvider>();
+            IRazorViewEngine engine = context.HttpContext.RequestServices.GetService<IRazorViewEngine>();
+            ITempDataProvider tempDataProvider = context.HttpContext.RequestServices.GetService<ITempDataProvider>();
+
+            context.HttpContext.AllowSynchronousIO();
 
             var actionContext = new ActionContext(context.HttpContext, context.HttpContext.GetRouteData(), new ActionDescriptor());
 
-            using (TextWriter textWriter = context.WriterFactory(context.HttpContext.Response.Body, selectedEncoding))
-            {
```

Comment:
main changes to this file was un-nesting usings, and adding `AllowSynchronousIO();`